manoseimas/mps_v2/static/components/filter_component.jsx

```diff
@@ -68,20 +70,33 @@ var SortableList = React.createClass({
   },
 
   selectFilter: function(option) {
-    this.setState({filter_selected: option, current_page: 1});
+    this.setState({
+        filter_selected: option,
+        current_page: 1
+    });
+    $.scrollTo('.sort-keys', 100, {offset: -50});
+  },
+
+  selectSubtab: function(tab, option) {
+    this.setState({
+        subtab_selected: option,
+        current_page: 1
+    });
     $.scrollTo('.sort-keys', 100, {offset: -50});
+
+    this.props.sidebar_subtabs.callback(tab, option);
   },
 
   render: function() {
+    var show_sidebar = Boolean(this.props.sidebar_filter) | Boolean(this.props.sidebar_subtabs);
     var sortkeys = this.props.keys,
         self = this,
         slice_from = 0,
         slice_to = this.state.current_page*this.state.items_per_page,
         current_page = this.state.current_page,
-        elementListWidth = (this.props.sidebar_filter) ? 14 : 16,
+        elementListWidth = show_sidebar ? 14 : 16,
         elementListWidthClass = num_to_word(elementListWidth),
-        filtered_items = this.state.items,
-        showSidebar;
+        filtered_items = this.state.items;
```

Comment:
You've removed `var showSidebar` and added `var show_sidebar`.

There's code inside this function that assigns to `showSidebar`.  Which is now a global variable.  Do you really want that?